src/test/java/com/example/rets_api/repository/PropertyRepositoryJPATest.java

```diff
@@ -42,8 +45,8 @@ public void should_store_a_Property() {
     @Test
     public void should_store_a_PropertyAndSchoolAndRoomAndViewData() {
```

Comment:
Please change the name to should_store_a_Property_With_All_Nested_Fields

Reply:
done